### PR TITLE
add compat for core refactoring

### PR DESCRIPTION
**Pull request by obestwalter**
Added a simple import compat layer - mainly for local development and tests - this can be removed once changes in core are merged.

### CODE REVIEW

File: tests/test_z_cmdline.py [hunk 1]
```diff
@@ -934,9 +934,13 @@ def test_exit_code(initproj, cmd, exit_code, mocker):
     initproj("foo", filedefs={'tox.ini': tox_ini_content})
     cmd()
     if exit_code:
+        try:
+            from tox.exc import exit_code_str
+        except ImportError:
+            from tox import _exit_code_str as exit_code_str
```
Comment (rpkilby) [inline tests/test_z_cmdline.py]: Should this occur before the `if exit_code` block? `exit_code_str` is not defined if entering the else block.

**Review by rpkilby — CHANGES_REQUESTED** [review]

**Comment by rpkilby** [comment]
Hi @obestwalter - I'm not following entirely. I assume this compat is related to the spring cleaning PR? And I'm assuming that the end state would be that `tox._exit_code_str` is being moved to the public API as `tox.exit_code_str`?

Do you need these changes merged for your own testing purposes, or is this just keeping tox-venv's test suite up-to-date? If it's the latter, I usually just update the tests once a new version of tox is released.

**Comment by rpkilby** [comment]
~~Also, looking at the spring cleaning PR, should this be `tox.exc.exit_code_str`?~~ 

nvm, missed that `exit_code_str` was imported from `tox.exc`.

**Comment by obestwalter** [comment]
Hi @rpkilby - sorry for not referencing that here. It's mainly for local dev, to work with different versions of of tox until it's moved over. Not even strictly necessary, but handy in the meantime?

**Comment by rpkilby** [comment]
That makes sense. I'll review once the test is fixed up.

**Comment by obestwalter** [comment]
whoops, that was a silly one :)

**Comment by rpkilby** [comment]
Hi @obestwalter. Do you need these changes merged, or is it sufficient if they live on a branch?

**Comment by obestwalter** [comment]
Ho @rpkilby - changes in core are merged, so no need any more for this I think :)

**Pull request closed without merging**